LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/snap-after-relayout/focus-element-no-snap.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<title>Resnap to focused element after relayout</title>
```

Comment:
Can you export this to WPT? `export-w3c-test-changes -b 249354 -g HEAD -c --no-linter`

Curious to see cross browser results for this test.

Reply:
https://github.com/web-platform-tests/wpt/pull/38151

Reply:
Seems to pass in other browsers 👍 

A bot should approve it when your change lands in WebKit.